dubbo-config/dubbo-config-spring/pom.xml

```diff
@@ -120,7 +120,6 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
```

Comment:
Now spring module still use `<version>4.12</version>` , it's a special usage . Please retain it